src/client/src/apps/MainRoute/widgets/analytics/analyticsService.test.ts

```diff
@@ -25,10 +25,15 @@ describe('AnalyticsService getAnalyticsStream', () => {
         cold<PositionsRaw>(actionLifetime, actionReference),
       )
       const serviceClient = new MockServiceStubWithLoadBalancer(createStreamOperation$)
-      const analyticsService = new AnalyticsService(serviceClient)
+      const analyticsService = new AnalyticsService(serviceClient as ServiceStubWithLoadBalancer)
```

Comment:
you will see that such explicit casting is done in a lot of tests now - this is the quickest way to fix that I found for now